src/component/feature/form/strategy-input-percentage.jsx

```diff
@@ -8,11 +9,19 @@ const labelStyle = {
     fontSize: '12px',
 };
 
-export default ({ name, value, onChange }) => (
+const InputPercentage = ({ name, value, onChange }) => (
     <div style={{ marginBottom: '20px' }}>
         <div style={labelStyle}>
             {name}: {value}%
         </div>
         <Slider min={0} max={100} defaultValue={value} value={value} onChange={onChange} label={name} />
```

Comment:
off topic: defaultValue and value should not be used at the same time. Input fields are either controller or uncontrolled, ref https://reactjs.org/docs/forms.html#controlled-components